src/views/Instance/styleCytoViz.js

```diff
@@ -25,12 +25,22 @@ export const getDefaultSelectedEdgeStyle = (theme) => ({
   ...getDefaultEdgeStyle(theme),
   width: EDGE_SELECTED_WIDTH,
   'line-color': EDGE_SELECTED_COLOR,
+  'target-arrow-color': EDGE_SELECTED_COLOR,
+  'target-arrow-shape': 'triangle',
 });
-export const getDefaultOutEdgeStyle = (theme) => ({
+export const getDefaultInEdgeStyle = (theme) => ({
   ...getDefaultEdgeStyle(theme),
   width: EDGE_SELECTED_WIDTH,
   'line-color': EDGE_SELECTED_COLOR,
   'line-style': 'dashed',
+  'target-arrow-color': EDGE_SELECTED_COLOR,
+  'target-arrow-shape': 'triangle',
+});
+export const getDefaultHiddenEdge = (theme) => ({
+  visibility: 'hidden',
+});
+export const getDefaultHiddenNode = (theme) => ({
+  visibility: 'hidden',
```

Comment:
To confirm with THU but can we use only one function like getDefaultHiddenStyle because both functions do the same thing ?